log4rs.yaml

```diff
@@ -33,6 +33,20 @@ appenders:
         base: 1
     encoder:
       kind: json
+  cache:
```

Comment:
In other services that use redis (like svc-telemetry) we have a `backend_requests` appender.

Reply:
https://github.com/aetheric-oss/svc-cargo/pull/43/commits/5eccc2bc4b4335866a129e31c449d470990edf83